platform-validation/scripts/aziot-compatibility.sh

```diff
@@ -553,6 +553,30 @@ check_shared_library_dependency_display_util() {
     esac
 }
 
+check_package_manager() {
+    not_found=0
+    package_managers="apt-get dpkg yum rpm"
```

Comment:
we should add dnf to the list as well, RHEL supports dnf

Reply:
rpm is similar to dpkg in a sense that it wont install 
so the order should be
apt-get dnf yum dpkg rpm {Check for Package Management Systems that install dependencies, then check for package managers)